ios/brave-ios/Sources/Data/models/SessionWindow.swift

```diff
@@ -65,7 +65,7 @@ extension SessionWindow {
   }
   
   public static func from(windowId: UUID, in context: NSManagedObjectContext) -> SessionWindow? {
-    let predicate = NSPredicate(format: "\(#keyPath(SessionWindow.windowId)) == %@", windowId.uuidString)
+    let predicate = NSPredicate(format: "\(#keyPath(SessionWindow.windowId)) == %@", windowId as NSUUID)
```

Comment:
is this change supposed to be part of this PR?

Reply:
There are a number of iOS specific changes in this PR when I was fixing unit tests, so yeah